sdk/trace/src/main/scala/org/typelevel/otel4s/sdk/trace/SdkContextKeys.scala

```diff
@@ -48,9 +51,7 @@ object SdkContextKeys {
     *   val baggage: Option[Baggage] = context.get(SdkContextKeys.BaggageKey)
     * }}}
     */
-  val BaggageKey: Context.Key[Baggage] =
-    Context.Key
-      .unique[SyncIO, Baggage]("otel4s-trace-baggage-key")
-      .unsafeRunSync()
+  @uncheckedStable
+  def BaggageKey: Context.Key[Baggage] = SdkBaggageManager.BaggageKey
```

Comment:
Since the `Baggage` belongs to the common package, I think we should move `BaggageKey` to the `sdk-common` package.

Reply:
Maybe we can make `SdkBaggageManager.BaggageKey` public and call it a day.

Reply:
my concern is, both within the javadocs for `io.opentelemetry.context.ContextKey` and in its usage for `BaggageContextKey.KEY` and `SpanContextKey.KEY`, the key is always non-public (either `private` in the case of the javadoc or Java's package-private in the case of the aforementioned two keys). I understand that various sdk propagators use `SdkContextKeys.{SpanContext,Baggage}Key`, but I feel like it should be limited to that if not restricted further to only be used by the sdk span backend and `SdkBaggageManager`

Reply:
That makes sense, I already forgot these details 😅 
I think we can keep it as-is, then. Thanks for the update!